demo/features/0453-issue-credential.feature

```diff
@@ -125,7 +125,7 @@ Feature: RFC 0453 Aries agent issue credential
        | --public-did --multitenant             | --multitenant             | driverslicense | Data_DL_NormalizedValues |
 
 
-  @T003.1-RFC0453
+  @T003.1-RFC0453 @BBS
```

Comment:
I think this is too big a hammer — e.g. applied at too high a level.  I notice in the `steps` that `DID_METHOD_KEY` is imported with the `BLS` ones.  That tells me that the tests might be run with both DID Key (aka Ed25519 keys) and/or with BLS ones.  I think we need to be more surgical about flagging / removing BBS tests while still leaving the DID Key tests.

Reply:
Looking at making these more granular.

Reply:
I have removed the `@BBS` tags for the scenarios and instead expanded the underlying examples to include tests specifically for ED25519 and BLS. The BLS examples are marked with `@BBS` making the scenarios more granular. Only these will be skipped when tests are run with `-t ~@BBS` while the ED25519 tests will always run.